### PR TITLE
Gatsby documentation

### PR DESCRIPTION
**Pull request by gd2910**
## Summary of the changes

Documentation page created for using @ukic components in Gatsby.

To review changes please see 'Get Started -> Get Components -> Gatsby'

## Checklist

- [x] I have [manually accessibility tested](https://design.sis.gov.uk/accessibility/testing/manual-testing) any changes, if relevant.
- [x] I have raised this pull request against the [develop branch](https://github.com/mi6/ic-design-system/tree/develop)

### CODE REVIEW

File: src/content/structured/get-started/gatsby.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+npm install @ukic/react @ukic/fonts
+
+// using yarn
+rm package-lock.json
+yarn add @ukic/react @ukic/fonts
+```
+
+## Step Two
+
+You need to intstall an extra package to allow Gatsby to render the components server side.
```
Comment (ad9242) [inline src/content/structured/get-started/gatsby.mdx:30]:
```suggestion
Install the `gatsby-plugin-stencil` package to allow Gatsby to render the components server side.
```

File: src/content/structured/get-started/gatsby.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+To get the correct styling with the ICDS components, import the core CSS file.
+
+Add the following to the `index.jsx` file of your project.
+
+```jsx
+project / src / pages / index.jsx;
+
+import "@ukic/fonts/dist/fonts.css";
+import "@ukic/web-components/dist/core/core.css";
```
Comment (ad9242) [inline src/content/structured/get-started/gatsby.mdx:74]:
```suggestion
import "@ukic/react/dist/core/core.css";
```

**Review by ad9242 — CHANGES_REQUESTED** [review]

File: src/content/structured/get-started/gatsby.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+```
+
+## Step five
+
+To get the correct styling with the ICDS components, import the core CSS file.
+
+Add the following to the `index.jsx` file of your project.
+
+```jsx
+project / src / pages / index.jsx;
```
Comment (jd239) [inline src/content/structured/get-started/gatsby.mdx:71]: It's possible the developer is using the TS template for Gatsby perhaps say something 

`Add the following to the top level index file for your project` and remove the folder destination from the code example.

**Review by jd239 — CHANGES_REQUESTED** [review]
Looks good. Just a single comment :)

**Comment by gd2910** [comment]
Comments actioned and 'helpful links' edited to follow style and tone in the rest of the document

File: src/content/structured/get-started/gatsby.mdx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Install the `gatsby-plugin-stencil` package to allow Gatsby to render the components server side.
+
+```shell
+// using npm
+npm install --save gatsby-plugin-stencil
+```
+
+## Step three
+
+In your `gatsby-config.js` file (at the root of the project) add the following to the `module.exports`:
```
Comment (ad9242) [inline src/content/structured/get-started/gatsby.mdx:39]:
```suggestion
In your `gatsby-config.js` file (at the root of the project) add the following:
```
I don't think it needs the module.exports (I think that is something for Vue?)

File: src/content/structured/get-started/gatsby.mdx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+```jsx
+project / src / pages / index.jsx;
+
+import "@ukic/fonts/dist/fonts.css";
+import "@ukic/react/dist/core/core.css";
+```
+
+## Helpful links
+
+The Gatsby documentation contains more information on the [Stencil plugin](https://www.gatsbyjs.com/plugins/gatsby-plugin-stencil/) used, and further information can be found on [using Stencil in Gatsby](https://dev.to/brunnerlivio/use-stencil-with-gatsbyjs-1omo).
```
Comment (ad9242) [inline src/content/structured/get-started/gatsby.mdx:79]:
```suggestion
The Gatsby documentation contains more information on the [plugin used.](https://www.gatsbyjs.com/plugins/gatsby-plugin-stencil/) Further information can be found on [using Stencil in Gatsby](https://dev.to/brunnerlivio/use-stencil-with-gatsbyjs-1omo).
```

Currently reads to me like it is a Stencil plugin, so have tried to make it sound less like that

**Review by ad9242 — CHANGES_REQUESTED** [review]

File: src/content/structured/get-started/gatsby.mdx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+```jsx
+project / src / pages / index.jsx;
+
+import "@ukic/fonts/dist/fonts.css";
+import "@ukic/react/dist/core/core.css";
+```
+
+## Helpful links
+
+The Gatsby documentation contains more information on the [plugin used.](https://www.gatsbyjs.com/plugins/gatsby-plugin-stencil/) Further information can be found on [using Stencil in Gatsby](https://dev.to/brunnerlivio/use-stencil-with-gatsbyjs-1omo).
```
Comment (ad9242) [inline src/content/structured/get-started/gatsby.mdx:79]:
```suggestion
The Gatsby documentation contains more information on the [plugin used.](https://www.gatsbyjs.com/plugins/gatsby-plugin-stencil/) Further information can be found on [using Stencil web components in Gatsby](https://dev.to/brunnerlivio/use-stencil-with-gatsbyjs-1omo).
```

File: src/content/structured/get-started/gatsby.mdx [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+npm install @ukic/react @ukic/fonts
+
+// using yarn
+rm package-lock.json
+yarn add @ukic/react @ukic/fonts
+```
+
+## Step Two
+
+Install the `gatsby-plugin-stencil` package to allow Gatsby to render the components server side.
```
Comment (GCHQ-Developer-847) [inline src/content/structured/get-started/gatsby.mdx:30]:
```suggestion
Install the `gatsby-plugin-stencil` package to allow Gatsby to render the components server-side.
```

**Review by GCHQ-Developer-847 — CHANGES_REQUESTED** [review]

**Comment by jd239** [comment]
https://animated-fiesta-gvyom1k.pages.github.io/branches/215-add-documentation-on-how-to-use-components-in-gatsby/

File: src/content/structured/get-started/gatsby.mdx [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+title: "Gatsby"
+
+subTitle: "How to use the components in a Gatsby project."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/gatsby.mdx"
+---
+
+## Step one
+
+In the root of your project:
```
Comment (DBD324) [inline src/content/structured/get-started/gatsby.mdx:17]: What is this suggesting? 'In the root write ...' or 'In the root find ...'. Need to add a verb to add context to the instruction.

File: src/content/structured/get-started/gatsby.mdx [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+Import the component(s) in your Gatsby pages.
+
+```jsx
+import { IcComponent } from "@ukic/react";
+```
+
+## Step five
+
+To get the correct styling with the ICDS components, import the core CSS file.
```
Comment (DBD324) [inline src/content/structured/get-started/gatsby.mdx:66]: "To get the correct styling for the ICDS components, import the core CSS file."

File: src/content/structured/get-started/gatsby.mdx [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+Add the following to the top level index file of your project.
+
+```jsx
+import "@ukic/fonts/dist/fonts.css";
+import "@ukic/react/dist/core/core.css";
+```
+
+## Helpful links
+
+The Gatsby documentation contains more information on the [plugin used](https://www.gatsbyjs.com/plugins/gatsby-plugin-stencil/). Further information can be found on [using Stencil web components in Gatsby](https://dev.to/brunnerlivio/use-stencil-with-gatsbyjs-1omo).
```
Comment (DBD324) [inline src/content/structured/get-started/gatsby.mdx:77]: Link is non-dedscriptive. Describe the think you are linking to. 'Plugin used' is non descriptive and contextual. Instead use 'Gatsby StencilJS plug in' used.

**Review by DBD324 — CHANGES_REQUESTED** [review]
Added some comments on the page. Couple of small content style points.

File: src/content/structured/get-started/gatsby.mdx (hunk 10 above)
Comment (jd239) [inline src/content/structured/get-started/gatsby.mdx:77]:
```suggestion
The Gatsby documentation contains more information on [gatsby-plugin-stencil](https://www.gatsbyjs.com/plugins/gatsby-plugin-stencil/). Further information can be found on [using Stencil web components in Gatsby](https://dev.to/brunnerlivio/use-stencil-with-gatsbyjs-1omo).
```

**Review by jd239 — APPROVED** [review]

**Review by ad9242 — APPROVED** [review]

**Pull request merged**